pymc_marketing/model_builder.py

```diff
@@ -341,6 +341,16 @@ def default(x):
             default=default,
         )
 
+        attrs["dag"] = (
+            json.dumps(self.dag, default=default) if hasattr(self, "dag") else "None"
+        )
+        attrs["treatment_nodes"] = (
+            self.treatment_nodes if hasattr(self, "treatment_nodes") else "None"
+        )
+        attrs["outcome_node"] = (
+            self.outcome_node if hasattr(self, "outcome_node") else "None"
+        )
```

Comment:
Thought we moved this to the mmm class. This shouldn't be in the model_builder

Reply:
Good catch, you did and was not removed.